package.json

```diff
@@ -9,7 +9,7 @@
     "@cypress/react": "^5.12.4",
     "@fortawesome/fontawesome-free": "^6.2.0",
     "bulma": "^0.9.4",
-    "classnames": "^2.3.1",
+    "classnames": "^2.3.2",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.